documentation/MatchMakerFramework.md

```diff
@@ -7,7 +7,7 @@ A locally running component, with the following responsiblities:
 
 ## Configuration
 
-Like all other Kettle apps, the Matchmaker Framework has a config file. Generally, the Matchmaker Framework configs are located in the `configs` folder of the matchMakerFramework component. The declaration of what matchmakers are available to the Matchmaker Framework, and where to find them should be set under the `matchMakers` option of the `gpii.matchMakerFramework`. Each entry should have the name of the mathcmaker as key, and then the value of an object which at least contains a `url`.  For example, in the following config file, two matchmakers are listed to be available. The "flat" matchmaker, located at the url http://localhost:8081 and the "RuleBased" one at the url http://localhost:8078.
+Like all other Kettle apps, the Matchmaker Framework has a config file. Generally, the Matchmaker Framework configs are located in the `configs` folder of the matchMakerFramework component. The declaration of what matchmakers are available to the Matchmaker Framework, and where to find them should be set under the `matchMakers` option of the `gpii.matchMakerFramework`. Each entry should have the name of the mathcmaker as key, and then the value of an object which at least contains a `url`.  For example, in the following config file, two matchmakers are listed to be available. The "default" matchmaker, located at the url http://localhost:8081 and the "RuleBased" one at the url http://localhost:8078.
```

Comment:
We need to plan for purely local matchmakers, preparing for the day when we no longer deploy components which make communications over localhost. So in practice, a matchmaker definition should really be a component definition (e.g. a dataSource or some other lightweight interface), or an IoC reference to one, rather than simply a bare URL.